server/routes/feed/[feed_type]/project/[id].js

```diff
@@ -0,0 +1,71 @@
+import { Feed } from 'feed'
+import { renderString } from '~/helpers/parse'
+
+export default defineEventHandler(async (event) => {
+  const config = useRuntimeConfig()
+  const API_URL = config.apiBaseUrl
+  const WEBSITE_URL = config.public.siteUrl
+
+  const projectInformation = await $fetch(API_URL + 'project/' + event.context.params.id)
+  const projectVersions = await $fetch(API_URL + 'project/' + event.context.params.id + '/version')
+  const projectTeam = await $fetch(API_URL + 'project/' + event.context.params.id + '/members')
+
+  let featuredImage = projectInformation.gallery.filter((image) => image.featured)[0]
+
+  if (featuredImage) {
+    featuredImage = featuredImage.url
+  }
+
+  const feed = new Feed({
+    title: projectInformation.title,
+    id: projectInformation.id,
+    description: renderString(projectInformation.description),
+    generator: 'Modrinth',
+    link: WEBSITE_URL + `/${projectInformation.project_type}/${projectInformation.id}`,
+    language: 'en',
+    updated: new Date(projectInformation.updated),
+    favicon: projectInformation.icon_url ?? 'https://cdn.modrinth.com/placeholder.png',
+    image: featuredImage ?? undefined,
+  })
+
+  projectVersions.forEach((version) => {
+    feed.addItem({
+      title: `New Version Released: ${version.name}`,
+      id: `release-${version.id}`,
+      link:
+        WEBSITE_URL +
+        `/${projectInformation.project_type}/${projectInformation.id}/version/${version.id}`,
+      content:
+        `This version is for ${version.game_versions.join(', ')}<br>` +
```

Comment:
This should probably also list the version's loaders (unless this already shows this?)
Plus, this should also show the version channel so that people could actually filter out alpha versions